dookug-document/dookug-document-service/src/main/java/hu/icellmobilsoft/dookug/document/service/repository/TemplateRepository.java

```diff
@@ -39,6 +39,10 @@ public interface TemplateRepository extends EntityRepository<Template, String>,
 
     @Query("SELECT t FROM Template t WHERE t.name = ?1 AND t.language=?2 AND "
             + " (?3 BETWEEN t.validityStart AND t.validityEnd OR (t.validityStart < ?3 AND t.validityEnd is null))")
-    Template findByNameAndValidity(String templateName, String templateLanguage, OffsetDateTime validityDate);
+    Template findByNameLanguageAndValidity(String templateName, String templateLanguage, OffsetDateTime validityDate);
+
+    @Query("SELECT t.id FROM Template t WHERE t.name = ?1 AND t.language=?2 AND "
+            + " (?3 BETWEEN t.validityStart AND t.validityEnd OR (t.validityStart < ?3 AND t.validityEnd is null))")
```

Comment:
`?3 BETWEEN t.validityStart AND t.validityEnd`
ezen a ponton nem száll el, ha a t.validityEnd az null?

Reply:
a teszt DBben az mindenhol null jelenleg